core/amber/.env

```diff
@@ -0,0 +1 @@
+OPENAI_API_KEY= " "
```

Comment:
Can ".env" file have a name, like we include some meaningful info in the filename?